e2e_deployment_files/long_haul_deployment.template.json

```diff
@@ -411,7 +411,7 @@
               }
             },
             "settings": {
-              "image": "<Container_Registry>/microsoft/azureiotedge-metrics-collector:<Build.BuildNumber>-linux-<Architecture>",
+              "image": "<Container_Registry>/microsoft/azureiotedge-test-metrics-collectorr:<Build.BuildNumber>-linux-<Architecture>",
```

Comment:
we missed one, but I think this is the last typo

Reply:
Fixed.